types/wordpress__data-controls/index.d.ts

```diff
@@ -53,9 +52,9 @@ export function apiFetch(options: APIFetchOptions): unknown;
  * @returns An object for registering the default controls with the store.
  */
 export const controls: {
-    API_FETCH: (action: Action) => Promise<any>;
-    DISPATCH: (action: Action) => void;
-    SELECT: (action: Action) => any;
+    API_FETCH: (action: any) => Promise<any>;
+    DISPATCH: (action: any) => void;
+    SELECT: (action: any) => any;
```

Comment:
The shape of this object type [doesn't seem to match the source at all anymore](https://github.com/WordPress/gutenberg/blob/%40wordpress/data-controls%402.28.0/packages/data-controls/src/index.js#L132). Should the available members be resynchronized?

Reply:
Hmm, you're right, however, it seems like it [never did](https://github.com/WordPress/gutenberg/blame/d2e301ed6dd1b0be42f1ca95736b39f735299292/packages/data-controls/src/index.js)... I'll take a look at what I can do with this.

Reply:
Resolved by #65008